scripts/page_browse.js

```diff
@@ -0,0 +1,259 @@
+    }
+    return retVal;
+};
+
+// Construct the buttons for horizontal/vertical split
+// and return a splitter variable.
+var viewSplitter = function(container) {
+    const textImageSplitID = "text_image_split";
+    const splitPercentID = "split_percent";
+    let splitter = splitControl();
+
+    // stored value is "horizontal" or "vertical"
+    let splitDirString = localStorage.getItem(textImageSplitID);
+    if(!splitDirString) {
+        splitDirString = "horizontal";
+    }
+    // splitDirection is a value defined in splitter
+    let splitDirection = (splitDirString === "horizontal") ? splitter.DIRECTION.HORIZONTAL : splitter.DIRECTION.VERTICAL;
+
+    let splitPercent = localStorage.getItem(splitPercentID);
+    if(!splitPercent) {
+        splitPercent = 50;
+    }
+
+    let mainSplit = splitter.setup(container, {splitDirection: splitDirection, splitPercent: splitPercent});
+
+    let vSplitImage = $("<img>", {src: proofIntData.buttonImages.imgVSplit});
+    let vSwitchButton = $("<button>", {type: 'button', class: 'img-button', title: proofIntData.strings.switchVert}).append(vSplitImage);
+    let hSplitImage = $("<img>", {src: proofIntData.buttonImages.imgHSplit});
+    let hSwitchButton = $("<button>", {type: 'button', class: 'img-button', title: proofIntData.strings.switchHoriz}).append(hSplitImage);
+    // The splitter could be laid out before images are loaded so that when
+    // images appear some controls could be pushed out of view
+    vSplitImage.on("load", mainSplit.reLayout);
+    hSplitImage.on("load", mainSplit.reLayout);
+
+    function setSplitControls(splitDirection) {
+        if (splitDirection === splitter.DIRECTION.VERTICAL) {
+            hSwitchButton.show();
+            vSwitchButton.hide();
+        } else {
+            hSwitchButton.hide();
+            vSwitchButton.show();
+        }
+    }
+
+    function changeSplit(splitDirection) {
+        mainSplit.setSplit(splitDirection);
+        setSplitControls(splitDirection);
+        splitDirString = (splitDirection === splitter.DIRECTION.HORIZONTAL) ? "horizontal" : "vertical";
+        localStorage.setItem(textImageSplitID, splitDirString);
+    }
+
+    vSwitchButton.click(function () {
+        changeSplit(splitter.DIRECTION.VERTICAL);
+    });
+
+    hSwitchButton.click(function () {
+        changeSplit(splitter.DIRECTION.HORIZONTAL);
+    });
+
+    setSplitControls(splitDirection);
+
+    mainSplit.dragEnd.add(function (percent) {
+        localStorage.setItem(splitPercentID, percent);
+    });
+
+    return {
+        mainSplit: mainSplit,
+        buttons: [vSwitchButton, hSwitchButton],
+    };
+};
+
+// Construct the image sizing controls.
+var imageControl = function() {
+    const imagePercentID = "image_percent";
+    let percent = localStorage.getItem(imagePercentID);
+    if(!percent) {
+        percent = 100;
+    }
+
+    let percentInput = $("<input>", {type: 'number', min: '1', max: '999', value: percent});
```

Comment:
If I change the image % number and hit Enter it doesn't apply the new image size, but rather submits the form and wants me to enter a new Project ID.

Reply:
I've rearranged the controls to avoid this and improved the way it works.